cmd/marketplace-operator/main.go

```diff
@@ -32,8 +32,9 @@ func main() {
 		logrus.Fatalf("failed to get watch namespace: %v", err)
 	}
 	resyncPeriod := time.Duration(5) * time.Second
-	logrus.Infof("Watching %s, %s, %s, %d", resource, catalogSourceConfigKind, namespace, resyncPeriod)
-	sdk.Watch(resource, catalogSourceConfigKind, namespace, resyncPeriod)
+	logrus.Infof("Watching %s, %s, %s", resource, catalogSourceConfigKind, namespace)
+	// Don't resync for the short term
```

Comment:
Probably a reason on why we don't resync along with interval(along with github issue) would help someone looking at the code.